pkg/beyla/config.go

```diff
@@ -222,12 +227,17 @@ func (c *Config) Validate() error {
 			" purposes, you can also set BEYLA_NETWORK_PRINT_FLOWS=true")
 	}
 
+	if c.Printer.Enabled() && c.JSONPrinter.Enabled() {
+		return ConfigError("print_traces and print_traces_json are mutually exclusive")
+	}
+
 	if c.Enabled(FeatureAppO11y) && !c.Noop.Enabled() && !c.Printer.Enabled() &&
 		!c.Grafana.OTLP.MetricsEnabled() && !c.Grafana.OTLP.TracesEnabled() &&
 		!c.Metrics.Enabled() && !c.Traces.Enabled() &&
-		!c.Prometheus.Enabled() {
+		!c.Prometheus.Enabled() && !c.JSONPrinter.Enabled() {
 		return ConfigError("you need to define at least one exporter: print_traces," +
-			" grafana, otel_metrics_export, otel_traces_export or prometheus_export")
+			" print_traces_json,grafana, otel_metrics_export, otel_traces_export or" +
```

Comment:
Small nit here, please add a <space> after `print_traces_json,`

Reply:
Oh, missed that, good catch! Thanks